src/python/fuchsia_util/__init__.py

```diff
@@ -0,0 +1,13 @@
+# Copyright 2019 Google LLC
```

Comment:
For discussion:

It seems better to include these scripts with the build, rather than copying them to CF as-is since the Fuchsia SDK is unstable. That way, for each build we should be guaranteed to have a set of scripts that work. This is important for regression task etc where we do bisects.

We're also in the midst of doing PY2->PY3, and copying things here may complicate things. Do these scripts work with PY3 already?

If we go with that, we also can't import the scripts as-is, because importing is fairly permanent and we shouldn't try to reimport if we set up a build for another revision. We may need something like a stable CLI interface here for the necessary functionality.

Reply:
Discussed during meeting today.

Consensus seems to be:

* we're hard-forking these scripts from Fuchsia, feel free to request changes s.t. they meet Fuchsia's style guide

* we agreed that the Fuchsia team will be responsible for validating that "whatever is placed in the build bucket" is run-able via these scripts

* we accept the risk that, should we need to change/modify these scripts (which we're hoping is rather unlikely!), performing bisects for older builds will become either impossible or require a cumbersome hack

Finally, I *think* these scripts work with Python 3, but I'm not sure how to test such a thing.  Guidance would be very appreciated!  :D

Reply:
>we accept the risk that, should we need to change/modify these scripts (which we're hoping is rather unlikely!), performing bisects for older builds will become either impossible or require a cumbersome hack

We also brought up the idea of having a minimal "input pusher" utility with every build for this purpose, since that's the part that is most likely to change. Did we reach a decision on whether or not to do this? @nopsledder